pkg/local_object_storage/engine/tree.go

```diff
@@ -115,3 +115,21 @@ func (e *StorageEngine) TreeGetChildren(cid *cidSDK.ID, treeID string, nodeID pi
 	}
 	return nil, err
 }
+
+// TreeGetOpLog implements the pilorama.Forest interface.
+func (e *StorageEngine) TreeGetOpLog(cid *cidSDK.ID, treeID string, height uint64) (pilorama.Move, error) {
+	var err error
+	var lm pilorama.Move
+	for _, sh := range e.sortShardsByWeight(cid) {
+		lm, err = sh.TreeGetOpLog(cid, treeID, height)
+		if err != nil {
+			e.log.Debug("can't put node in a tree",
```

Comment:
wrong err message?

also, i think `height` may be added to the message

Reply:
Fixed